packages/vscode-apollo/src/extension.ts

```diff
@@ -3,90 +3,63 @@ import {
   window,
   workspace,
   ExtensionContext,
-  ViewColumn,
-  WebviewPanel,
   Uri,
   ProgressLocation,
   DecorationOptions,
   commands,
-  QuickPickItem
+  QuickPickItem,
+  Disposable,
+  OutputChannel
 } from "vscode";
-import {
-  LanguageClient,
-  LanguageClientOptions,
-  ServerOptions,
-  TransportKind
-} from "vscode-languageclient";
-
 import StatusBar from "./statusBar";
+import { getLanguageServerClient } from "./languageServerClient";
+import { LanguageClient } from "vscode-languageclient";
 import {
-  printStatsToClientOutputChannel,
-  printNoFileOpenMessage
+  printNoFileOpenMessage,
+  printStatsToClientOutputChannel
 } from "./utils";
-const { version, referenceID } = require("../package.json");
+
+const { version } = require("./package.json");
```

Comment:
Looks like this path got messed up in rebase or something :) 

```suggestion
const { version } = require("../package.json");
```